src/Categories/Category/Monoidal/Star-Autonomous.agda

```diff
@@ -0,0 +1,35 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Categories.Category
+open import Categories.Category.Monoidal.Core using (Monoidal)
+open import Categories.Category.Monoidal.Symmetric
+
+module Categories.Category.Monoidal.Star-Autonomous {o ℓ e} {C : Category o ℓ e} (M : Monoidal C) where
+
+open import Level
+
+open import Categories.Category.Product
+open import Categories.Functor
+open import Categories.Functor.Properties
+open import Categories.NaturalTransformation.NaturalIsomorphism
+open import Categories.Morphism C
+open import Categories.Functor.Hom
+open Hom C
+
+open Category C renaming (op to Cᵒᵖ) hiding (id)
+open Monoidal M
+open Functor ⊗ renaming (op to ⊗ₒₚ)
+
+open import Relation.Binary.Structures
+
+record Star-Autonomous : Set (levelOfTerm M) where
+  field
+    symmetric : Symmetric M
+    F⁻¹ : Functor Cᵒᵖ C
+
+  open Functor F⁻¹ renaming (op to F⁻¹ₒₚ ; F₀ to F⁻¹₀)
+
+  field
+    FFF⁻¹ : FullyFaithful F⁻¹
+    A≃A⁻¹⁻¹ : id ≃ (F⁻¹ ∘F F⁻¹ₒₚ)
```

Comment:
I think this would make more sense being called `A≃A**`, matching conventional notation. The star is a dual rather than an inverse. Similar comment for `𝒞[A⊗B,C⁻¹]≃𝒞[A,B⊗C⁻¹]`.

Reply:
@laMudri is there any way to write superscript `*` in Agda?

Reply:
Oh, I forgot that there are fonts where ASCII `*` doesn't appear as a superscript already.

Reply:
I've used normal `*` in the code. What do you think?

Reply:
It always looked fine for me. The question is whether anyone else would object. I want to say “eh, it'll be fine”, but I can't really judge that while I always use fonts that put `*` in the superscript position.

Reply:
Okay! Let's see what other people say :)

Reply:
It's fine with me.

Reply:
Great! :smile: